internal/recipe_l1.go

```diff
@@ -62,8 +62,16 @@ func (l *L1Recipe) Apply(ctx *ExContext, artifacts *Artifacts) *Manifest {
 		// we are going to use the cl-proxy service to connect the beacon node to two builders
 		// one the 'el' builder and another one the remote one
 		elService = "cl-proxy"
+
+		primaryBuilder := "el"
+		if l.useNativeReth {
```

Comment:
This should be happening already because the docker runtime should resolve this reference.